kikaha-modules/kikaha-rocker/tests/kikaha/rocker/RockerTest.java

```diff
@@ -5,6 +5,7 @@
 import kikaha.config.Config;
 import kikaha.core.test.HttpServerExchangeStub;
 import kikaha.core.test.KikahaRunner;
+import lombok.val;
```

Comment:
About lombok's `val`: I'm advocating people to avoid it on Kikaha.
I'm expecting to use `val` (or even `var`) on 2.2.x and further versions, but the current code base does not use it. I think the best think to keep the code concise it make all variable declarations following the same pattern.

But, please, I also would like to rear your opinion about it. ;) What do you think..?